cli/test/test_common.py

```diff
@@ -0,0 +1,54 @@
+import os
+import sys
+import shutil
+import contextlib
+import subprocess
+from git import Repo
+
+os.environ['test_repo_path'] = '/tmp/mypaper'
+os.environ['CI'] = 'false'
+
+pushstack = list()
+
+
+def pushdir(dirname):
```

Comment:
there's no need to have `pushdir` and `popdir` as the GitPython library can handle that. We do that on the shell scripts because we assume we run on the shell, but now that we are running these tests in Python, we can discard some shell-specific functionality